department-of-veterans-affairs/vets-website
# CT-103: Staging modal persists on browser navigation #9408

## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9408

## Testing done
1.  http://localhost:3001/gi-bill-comparison-tool/
2. search for a institution or select vet tec
3. Select a result
4. If the "Search Results" breadcrumb is not present on profile page, repeat 1-3 until it does appear
5. Open a modal
6. Use browser back button to get to Search Results, notice modal closes
7. Select a result
8. Use browser back button, observe no new errors in developer console in browser

## Screenshots


## Acceptance criteria

## Definition of done
